openedx/core/djangoapps/lang_pref/middleware.py

```diff
@@ -27,9 +29,12 @@ def process_request(self, request):
         If a user's UserPreference contains a language preference, use the user's preference.
         Save the current language preference cookie as the user's preferred language.
         """
-        cookie_lang = request.COOKIES.get(settings.LANGUAGE_COOKIE, None)
+        cookie_lang = request.COOKIES.get(settings.LANGUAGE_COOKIE_NAME, None)
         if cookie_lang:
             if request.user.is_authenticated:
+                # DarkLangMiddleware has already taken care of this so don't change anything
```

Comment:
```suggestion
                # DarkLangMiddleware will take care of this so don't change anything
```
Small wording suggestion -- DarkLangMiddleware comes later during the request phase.